src/android/HockeyApp.java

```diff
@@ -67,15 +67,11 @@ private Bitmap getBitmap() {
             long start = System.currentTimeMillis();
             webView.getPluginManager().postMessage(XWALK_SCREENSHOT_CAPTURE_MSG, this);
             try {
-                // Can't take more than 5 seconds to get a screenshot
-                long wait = XWALK_SCREENSHOT_WAIT_MS;
-                while (bitmap == null && wait > 0) {
-                    synchronized (monitor) {
-                        monitor.wait(wait);
-                    }
-                    wait = XWALK_SCREENSHOT_WAIT_MS - (System.currentTimeMillis() - start);
+                synchronized (monitor) {
+                    monitor.wait(XWALK_SCREENSHOT_WAIT_MS);
                 }
-            } catch (InterruptedException e) {
+            } catch (InterruptedException ex) {
```

Comment:
Actually, can we print the stack trace before re-interrupting. Just so we have better debug information if it ever happens.